tests/integration/aws/iot/single-rule-single-function/tests.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const path = require('path');
+const expect = require('chai').expect;
+const Utils = require('../../../../utils/index');
+
+describe('AWS - IoT: Single topic with single function', () => {
```

Comment:
This description should be `AWS - IoT: Single rule with single function` (so that all the names are following the same format)